src/main/pages/che-7/overview/proc_finding-che-cluster-url-using-openshift-4-cli-tools.adoc

```diff
@@ -1,22 +1,33 @@
 [id="finding-{prod-id-short}-cluster-url-using-openshift-4-cli-tools_{context}"]
-= Finding {prod-short} cluster URL using OpenShift 4 CLI tools
+= Finding {prod-short} cluster URL using the OpenShift 4 CLI
 
-This section describes how to {prod-short} cluster URL using OpenShift 4 CLI tool.
+This section describes how to obtain the {prod-short} cluster URL using the OpenShift 4 CLI (command line interface). The URL can be retrieved from the OpenShift logs or from the `checluster` Customer Resource.
 
-[discrete]
-== Prerequisites
+.Prerequisites
+* {prod-short} is deployed in a OpenShift cluster. This can be validated by osbserving Openshift logs. See xref:viewing-the-state-of-the-{prod-id-short}-cluster-deployment-using-openshift-4-cli-tools_{context}[].
+* User is located in a {prod-short} installation namespace.
 
-* View the logs of the {prod-short} Operator. See xref:viewing-the-state-of-the-{prod-id-short}-cluster-deployment-using-openshift-4-cli-tools_{context}[].
+.Procedure
 
-[discrete]
-== Procedure
+. For retrieving a {prod-short} cluster URL from the `checluster` CR, run the following command:
++
+[options="nowrap",role=white-space-pre]
+----
+$ oc get checluster --output jsonpath='{.items[0].status.cheURL}'
+----
 
-* Wait for the logs to show the message `{prod} is now available at:` followed by {prod-short} URL:
+. The {prod-short} cluster URL is displayed.
++
+Alternatively, wait for the logs to show the message `{prod} is now available at:` followed by the {prod-short} cluster URL:
```

Comment:
I'm slightly confused about this part, I can't test the procedure rn myself, sorry. =( 

The user runs a command, the URL is displayed and then does the message `{prod} is now available at:` pops up? Don't you need to run `oc log` or smth to get the logs?